src/constants.ts

```diff
@@ -78,41 +78,37 @@ export enum MODIFIER_KEY {
  * Categories used to organised the shortcut dialog.
  * Shortcut name should match those obtained from the Blockly shortcut register.
  */
-export const SHORTCUT_CATEGORIES = {
+export const SHORTCUT_CATEGORIES: Record<
+  string,
+  // Also allow undo/redo. Document the non-keyboard-nav versions of others for better text
+  // because temporarily the name in the table is derived from these id-like names.
```

Comment:
This was a surprise! I looked for the "Toggle keyboard nav" text to improve it and was surprised it didn't exist as such. Hopefully #155 can address this.

Reply:
This entire file would have gone away quite a long time ago except that that refactor has never been as important as making functional changes, but you are correct to suspect that we'll likely be changing how all of these shortcuts are defined at some point in the near future.

(In the mean time, though: may I encourage you to try to keep comment lines to <= 80 chars, please?)

Reply:
Sure, 88374cefdd8cb8b663901f1d742d74c88c22c5b8